packages/kyodo-contracts/test/TestKyodoDAO.test.js

```diff
@@ -1,5 +1,5 @@
 import truffleAssert from 'truffle-assertions';
-import assertRevert from 'openzeppelin-solidity/test/helpers/assertRevert';
+import { shouldFail } from 'openzeppelin-test-helpers';
```

Comment:
👍 looks more descriptive now

Reply:
https://github.com/OpenZeppelin/openzeppelin-test-helpers/ seems to be published very recently and has no issues with transpilation within truffle